app/controllers/planning_applications_controller.rb

```diff
@@ -11,7 +11,16 @@ class PlanningApplicationsController < AuthenticationController
     with: :decision_notice_mail_error
 
   def index
-    @planning_applications = policy_scope(PlanningApplication.all)
+    all_applications = policy_scope(PlanningApplication.all)
+    if current_user.assessor?
+      @planning_applications = PlanningApplication.where(
```

Comment:
That looks great! Have you thought of moving that logic into a scope in the `planning_application.rb`?

Reply:
ah thanks Evangelos! Yes will do - I'm going to try and place this logic inside a scope now and hopefully it won't break the associaiton - this first WIP was just to try and get the logic to work :)

Reply:
Yeah that's not a problem at all! 😄